prow/config.yaml

```diff
@@ -4032,6 +4065,28 @@ periodics:
         requests:
           cpu: 4
 
+- name: ci-kubernetes-e2e-kubeadm-dind-ipv6
+  interval: 6h
+  agent: kubernetes
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+    - image: diverdane/e2e-kubeadm-dind:v20180814-829bf6a-dirty
+      args:
+      - "--repo=k8s.io/kubernetes=master"
+      - "--repo=github.com/leblancd/kubeadm-dind-cluster=dockernet-172.20.0.0"
```

Comment:
upstream does not work?

Reply:
I added a note in the PR description above. We had been using the upstream stable branch for the originally proposed GCE-based CI, but there were 3 problems we encountered when running this in a DinDinD configuration (in a Prow container). We have fixes proposed to K-D-C, but we're waiting for the fixes to get merged into master, and then have the stable branch updated to inherit the changes. The fixes are:
      * [K-D-C PR # 175](https://github.com/kubernetes-sigs/kubeadm-dind-cluster/pull/175)
      * [K-D-C PR # 185](https://github.com/kubernetes-sigs/kubeadm-dind-cluster/pull/185)
      * [K-D-C PR # 199](https://github.com/kubernetes-sigs/kubeadm-dind-cluster/pull/199)